docs/resources/organizations_policy.md

```diff
@@ -0,0 +1,69 @@
+---
+subcategory: "Organizations"
+---
+
+# huaweicloud_organizations_policy
+
+Manages an Organizations policy resource within HuaweiCloud.
+
+## Example Usage
+
+### Create service control policy
+
+```hcl
+resource "huaweicloud_organizations_policy" "scp_policy"{
+  name     = "test_policy_name"
+  contenet = "{\"Version\":\"5.0\",\"Statement\":[{\"Effect\":\"Deny\",\"Action\":[]}]}"
```

Comment:
it's better to use jsonencode function or heredoc
https://jeffbrown.tech/terraform-heredoc-jsonencode/

Reply:
done